locales/en-US/app.ftl

```diff
@@ -37,6 +37,7 @@ landingLegalNotice = By proceeding, you agree to the <a>Terms of Use</a> and <a>
 [[landingInstall]]
 landingInstallButton = Install the Test Pilot Add-on
 landingInstallingButton = Installing...
+landingEnablingButton = Enabling...
```

Comment:
Please use a single unicode `…` instead of `...`. Fixing the old string would be great

Reply:
@flodolo will I need to update the key for `landingInstallingButton` for this change?

Reply:
No, each locale should have their own rules for punctuation signs (English should always use the single unicode character)